wsi/drm/drmdisplay.h

```diff
@@ -171,6 +171,10 @@ class DrmDisplay : public PhysicalDisplay {
 
   void TraceFirstCommit();
 
+  uint32_t GetPreferedDisplayMode(size_t modes_size);
```

Comment:
Need to be private?

Reply:
No need to invoke them out of this class, so set them as private method